frontend/src/app/general/metadata/version/version.component.html

```diff
@@ -3,22 +3,21 @@
  ~ SPDX-License-Identifier: Apache-2.0
  -->
 
-<a>
-  <div class="wrapper" (click)="versionService.openReleaseNotes()">
-    <div
-      class="version"
-      *ngIf="versionService.changedVersion && versionService.version"
-    >
-      New version available (<span *ngIf="versionService.oldVersion"
-        >{{ versionService.oldVersion }}
-        <mat-icon class="mat-icon-position top left"
-          >arrow_right_alt</mat-icon
-        ></span
-      >{{ versionService.version.git.tag }})!
-    </div>
+<div class="wrapper">
+  <div
+    class="version"
+    *ngIf="versionService.changedVersion && versionService.version"
+  >
+    New version available (<span *ngIf="versionService.oldVersion"
+      >{{ versionService.oldVersion }}
+      <mat-icon class="mat-icon-position top left"
+        >arrow_right_alt</mat-icon
+      ></span
+    >{{ versionService.version.git.tag }})!
+  </div>
+  <a href="/docs/release-notes/" target="_blank">
```

Comment:
I would probably also place the `<a>' around the *New version available ...* `<div>` because otherwise you can't click on it.